docs/01-Chart-Configuration.md

```diff
@@ -257,6 +257,18 @@ afterBody | `Array[tooltipItem], data` | Text to render after the body section
 beforeFooter | `Array[tooltipItem], data` | Text to render before the footer section
 footer | `Array[tooltipItem], data` | Text to render as the footer
 afterFooter | `Array[tooltipItem], data` | Text to render after the footer section
+dataPoints | `Array[CustomTooltipDataPoint], data` | List of matching point informations.
+
+#### CustomTooltipDataPoint
```

Comment:
Alright, yes up to you we can merge the two interface together.

In my opinion, `x` and `y` properties are usefull. I am using them in several of my projects to be able to show individual tooltips at each points position.